### PR TITLE
Wizard title - make it optional prop

### PR DESCRIPTION
**Pull request by ashasunmsft**
#### Pull request checklist

- [x ] Include a change request file using `$ npm run change`

#### Description of changes

Making title element optional as SetupWizard design has no title

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8138)

### CODE REVIEW

**Comment by ashasun** [comment]
@xbtsw is added to the review. #Closed

**Comment by ashasun** [comment]
@Raghurk is added to the review. #Closed

**Comment by ashasun** [comment]
@timtong1982 is added to the review. #Closed

File: packages/dashboard/src/components/Wizard/Wizard.styles.ts [hunk 1]
```diff
@@ -8,9 +9,10 @@ export const getWizardStyles = (): IWizardStyles => {
     },
     subwayNavSection: {
       width: '303px',
-      marginLeft: '48px',
-      marginTop: '48px',
-      marginBottom: '48px'
+      paddingLeft: '48px',
+      paddingTop: '48px',
+      paddingBottom: '48px',
+      borderRight: `1px solid ${NeutralColors.gray40}`
```
Comment (sysofwan) [inline packages/dashboard/src/components/Wizard/Wizard.styles.ts]: should this use the theme color instead (i.e. theme.semanticColors.bodyDivider)

Comment (sysofwan) [reply packages/dashboard/src/components/Wizard/Wizard.styles.ts]: From the [mocks](https://www.figma.com/proto/CjZhXXZ20uYf4i5yxi2H4a5m/14-M365-Admin-Guidelines-Wizards?node-id=1%3A961&scaling=min-zoom), looks like there is no border right for setup wizard?

**Pull request merged**